docs/migration_guides/qi_migration.rst

```diff
@@ -0,0 +1,742 @@
+The following table summarizes the most common :class:`~qiskit.utils.QuantumInstance` settings and which
+primitives **expose a similar setting through their interface**:
+
+.. attention::
+
+    In some cases, a setting might not be exposed through the interface, but there might an alternative path to make
+    it work. This is the case for custom transpiler passes, which cannot be set through the primitives interface,
+    but pre-transpiled circuits can be sent if setting the option ``skip_transpilation=True``. For more information,
+    please refer to the API reference or source code of the desired primitive implementation.
+
+.. list-table::
+   :header-rows: 1
+
+   * - QuantumInstance
+     - Reference Primitives
+     - Aer Primitives
+     - Runtime Primitives
+     - Backend Primitives
+   * - Select ``backend``
+     - No
+     - No
+     - Yes
+     - Yes
+   * - Set ``shots``
+     - Yes
+     - Yes
+     - Yes
+     - Yes
+   * - Simulator settings: ``basis_gates``, ``coupling_map``, ``initial_layout``, ``noise_model``, ``backend_options``
+     - No
+     - Yes
+     - Yes
+     - No
+   * - Transpiler settings: ``seed_transpiler``, ``optimization_level``
+     - No
+     - No
+     - Yes (via ``options``) (*)
+     - Yes (via ``.set_transpile_options()``)
+   * - Set unbound ``pass_manager``
+     - No
+     - No
+     - No (but can ``skip_transpilation``)
+     - No (but can ``skip_transpilation``)
+   * - Set ``bound_pass_manager``
+     - No
+     - No
+     - No
+     - Yes
+   * - Set ``backend_options``: common ones were ``memory`` and ``meas_level``
+     - No
+     - No
+     - No (only ``qubit_layout``)
+     - No
+   * - Measurement error mitigation: ``measurement_error_mitigation_cls``, ``cals_matrix_refresh_period``,
+       ``measurement_error_mitigation_shots``, ``mit_pattern``
+     - No
+     - No
+     - Sampler default -> M3 (*)
+     - No
+   * - Job management: ``job_callback``, ``max_job_retries``, ``timeout``, ``wait``
+     - No
+     - No
+     - Sessions, callback (**)
+     - No
+
+
+(*) For more information on error mitigation and setting options on Runtime Primitives, visit
+`this link <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.options.Options.html#qiskit_ibm_runtime.options.Options>`_.
+
+(**) For more information on Runtime sessions, visit `this how-to <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/how_to/run_session.html>`_.
+
+Code examples
+=============
+
+.. dropdown:: Example 1: Circuit Sampling with Local Statevector Simulation
+    :animate: fade-in-slide-down
+
+    **Using Quantum Instance**
+
+    The only alternative for local simulations using the quantum instance was using an Aer Simulator backend.
+    Please note that ``QuantumInstance.execute()`` returned the counts bitstrings in hexadecimal format.
+
+    .. testcode::
```

Comment:
If you don't plan to run the tests (I see that you used the `SKIP` flag) you can use `.. code-block:: python` and `.. code-block:: text` for input and output respectively so you don't depend on doctest.

Reply:
I didn't know about `.. code-block:: text`, thanks!